package.json

```diff
@@ -76,6 +77,7 @@
     "eslint-plugin-jest": "26.2.2",
     "eslint-plugin-prettier": "4.0.0",
     "http-server": "^14.1.1",
+    "husky": "^8.0.1",
```

Comment:
Is it okay to use carets here? (`^`)

Reply:
I think `devDependencies` is ok because it is a local development mode. It is not included in the release build like `dependencies` is. What do you think?

Reply:
I will merge this, but let's sync regarding `devDependencies` and if we want to have fixed versions there.